bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -135,97 +135,73 @@ $BallSeen
-            INDIRECT_FREEKICK --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-            PENALTYKICK --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-            CORNER_KICK --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-            GOAL_KICK --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-            THROW_IN --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-        FINISHED --> @StandAndWait
-        NOT_ALLOWED_TO_MOVE --> @LookAtBall, @StandAndWait
-    DEFENSE --> $GameStateDecider
-        INITIAL --> @LookAtFieldFeatures, @StandAndWait
-        READY --> #PositioningReady
-        SET --> @StandAndWait
-        PLAYING --> $SecondaryStateDecider
-            NORMAL --> #DefenseBehavior
-            PENALTYSHOOT --> @LookAtBall, @StandAndWait
-            OVERTIME --> #DefenseBehavior
-            TIMEOUT --> @LookAtBall, @StandAndWait
-            DIRECT_FREEKICK --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-            INDIRECT_FREEKICK --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-            PENALTYKICK --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-            CORNER_KICK --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-            GOAL_KICK --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-            THROW_IN --> $SecondaryStateTeamDecider
-                OUR --> @LookAtBall, @StandAndWait
-                OTHER --> @LookAtBall, @StandAndWait
-        FINISHED --> @StandAndWait
-        NOT_ALLOWED_TO_MOVE --> @LookAtBall, @StandAndWait
-    PENALTY --> $GameStateDecider
-        INITIAL --> @LookAtFieldFeatures, @StandAndWait
-        READY --> #PositioningReady
-        SET --> @StandAndWait
-        PLAYING --> #PenaltyBehavior
-        FINISHED --> @StandAndWait
-        NOT_ALLOWED_TO_MOVE --> @LookAtBallPenalty, @StandAndWait
+    GOALIE --> $SecondaryStateDecider
+        PENALTYSHOOT --> $SecondaryStateTeamDecider
+            OUR --> $GameStateDecider
+                READY --> @GoToBall
+                PLAYING --> @KickBallDynamic
+                ELSE --> @LookAtBall, @StandAndWait
+            OTHER --> #GoalieBehavior
```

Comment:
Wouldn't the robot run to the ball if it perceives it as close enough?
I assume that would be against the rules?